tests/foreman/ui/test_contentview.py

```diff
@@ -1693,7 +1598,7 @@ def test_positive_delete_cv_promoted_to_multi_env(session, module_org):
     :Steps:
 
         1. Create a content view
-        2. Add a yum repo and a puppet module to the content view
+        2. Add a yum repo to the content view
```

Comment:
:+1: funny, no usage of puppet module but it was still written in the description :smile: